about.html

```diff
@@ -0,0 +1,94 @@
+    <link rel="stylesheet" href="style.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+    <link rel="stylesheet" href="style2.css">
+<body>
+    <header>
+        <ul class="nav nav justify-content-end">
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">My page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+    </header>
+    <div class="top">
+        <div class="top1">
+            <img alt="logo" src="Images/Logo.png">
+            <ul class="nav">
+                <li class="nav-item">
+                <a class="nav-link" href="index.html">Home</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="about.html">about</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Sponsors</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link active" href="#">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <div class="main">
+        <div class="overlay">
+            <div class="menu-overlay" onclick = "showMenu()">
+                <div class="menu">
+                    <div class="login">Login</div>
+                    <a class="menu-link" href="index.html">Home</a>
+                    <a class="menu-link" href="about.html">about</a>
+                    <a class="menu-link" href="#">Join</a>
+                    <a class="menu-link" href="#">Sponsors</a>
+                    <a class="menu-link" href="#">Contact</a>
+                </div>
+            </div>
+            <div class="navigation"  onclick = "showMenu()">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            
+
+            <section class="intro" id="Intro">
+                <h1 class="main-title">THE WORLD  FOOTBALL GOVERNING BODY (FIFA)<br> HAS SETTLED ON RWANDA TO HOST THE 73rd FIFA CONGRESS</h1>
+                <div class="intro-message">
+                    <p>The Fédération internationale de football association is the international governing body of association football, beach soccer, and futsal. It was founded in 1904 to oversee international competition among the national associations of Belgium, Denmark, France, Germany, the Netherlands, Spain, Sweden and Switzerland</p>
+                </div>
+            </section>
+        </div>
+    </div>
+    
```

Comment:
- [ ] You literally miss one section in your website, this section contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo 🙃. Please kindly add it 😁 😁 

Original design

<img width="873" alt="Screen Shot 2023-02-16 at 00 06 34" src="https://user-images.githubusercontent.com/74501523/219161672-c0c45317-64eb-4d4f-9e77-83370718ba22.png">